README.md

```diff
@@ -36,7 +36,7 @@ const getJwks = buildGetJwks({
 - `max`: Max items to hold in cache. Defaults to 100.
 - `ttl`: Milliseconds an item will remain in cache. Defaults to 60s.
 - `timeout`: Specifies how long it should wait to retrieve a JWK before it fails. The time is set in milliseconds. Defaults to 5s.
-- `allowedDomains`: Array of allowed domains. By default all domains are allowed.
```

Comment:
here,  I would keep the deprecated option and mention explicitly that it's been deprecated. meanwhile let's open an issue to remove this option altogether, including the code change which, at the current time, preserves compatibility

Reply:
Sounds good 👍 created an issue [here](https://github.com/nearform/get-jwks/issues/230).